app/presenters/human_time.rb

```diff
@@ -11,13 +11,13 @@ def initialize(time:, current_time: Time.current)
   def relative_start_time
```

Comment:
we are using this method in a few places -- double check that this usage makes sense in all places? Otherwise, let's add these extra strings in the place where you want them and interpolate `relative_start_time`